core/backend/src/Relationship.ts

```diff
@@ -188,7 +201,185 @@ export interface ElementDrivesElementProps extends RelationshipProps {
   priority: number;
 }
 
-/** A Relationship where one Element *drives* another Element
+/** A Relationship indicating that one Element *drives* another Element.
```

Comment:
This thorough documentation seems like it belongs in a dedicated article in /docs/learning.

Reply:
I'd prefer to wait until I (or someone else) has time to write a real example to go with it.

Reply:
Ok.